src/applications/gi/containers/LandingPage.jsx

```diff
@@ -177,6 +178,7 @@ export class LandingPage extends React.Component {
               {!isVetTecSelected(this.props.filters) && (
                 <KeywordSearch
                   label={searchLabel}
+                  searchOnAutcompleteSelection={landingPageCheck}
```

Comment:
```suggestion
                  searchOnAutcompleteSelection
```

I think just something like this is valid

Reply:
updated